torchci/pages/tritonbench/commit_view.tsx

```diff
@@ -61,6 +69,18 @@ export default function Page() {
       setGranularity(granularity);
     }
 
+    const benchmarkName: string =
+      (router.query.benchmarkName as string) ?? DEFAULT_TRITON_BENCHMARK_NAME;
+    if (benchmarkName !== undefined) {
+      setBenchmarkName(benchmarkName);
+    }
+
+    const deviceName: string =
+      (router.query.deviceName as string) ?? DEFAULT_DEVICE_NAME;
+    if (deviceName !== undefined) {
```

Comment:
How could it be undefined at this point if you have `?? DEFAULT_VALUE` line above?